lib/routes/devices-and-sessions.js

```diff
@@ -250,7 +250,7 @@ module.exports = (log, db, config, customs, push, pushbox, devices) => {
             spurious = false
           }
 
-          if (payload.availableCommands) {
+          if (payload.availableCommands && token.deviceAvailableCommands) {
```

Comment:
Good point.

Reply:
> And if so, would that lead to us inserting more rows into deviceCommands than we intended?

It might result in us overwriting rows more often than we need to, but I don't think it would have produced any additional rows.